### PR TITLE
feat: Adds appveyor build file

### LINKED ISSUE #54: Add Appveyor
Generic setup for testing on windows

### PR DESCRIPTION
**Pull request by d3viant0ne**
Adds an appveyor build file to `webpack-defaults`

- The service has been configured for the org - https://ci.appveyor.com/projects
- The service account has already been configured - https://github.com/webpack-contrib-appveyor
- Teams within contrib have been assigned the proper permissions to access the service.

Output ...

```
branches:
  only:
    - master

init:
  - git config --global core.autocrlf input

environment:
  matrix:
    - nodejs_version: 7
      webpack_version: 2.2.0
      job_part: test
    - nodejs_version: 6
      webpack_version: 2.2.0
      job_part: test
    - nodejs_version: 4.3
      webpack_version: 2.2.0
      job_part: test

install:
  - ps: Install-Product node $env:nodejs_version x64
  - npm install

build: off

matrix:
  fast_finish: true

before_test:
  - cmd: npm install webpack@^%webpack_version%

test_script:
  - node --version
  - npm --version
  - cmd: npm run appveyor:%job_part%
```

Closes #54

### CODE REVIEW

**Comment by sapegin** [comment]
Do we need someone else’s review or I can merge it?

**Comment by bebraw** [comment]
@d3viant0ne Do you have any changes in mind for this? I don't understand Appveyor well enough to comment.

**Comment by d3viant0ne** [comment]
I wanted to try this config out in a project before I merged it, to make sure it works the way I think it should.

File: src/tasks/appveyor.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          'master',
+        ],
+      },
+      init: [
+        'git config --global core.autocrlf input',
+      ],
+      environment: {
+        matrix: [
+          {
+            nodejs_version: '7',
```
Comment (michael-ciniawsky) [inline src/tasks/appveyor.js]: `7` => `8`  :)

Comment (d3viant0ne) [inline src/tasks/appveyor.js]: @michael-ciniawsky - Now configured to use the config for it's nodejs versions like the travis template.

**Pull request merged**